### ISSUE TITLE
Web - Overlay background colour is different

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#27353]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Open settings on staging
2. Open settings on newdot
3. Observe the overlay color

## Expected Result:
The overlay should be same as the newdot.

## Actual Result:
The overlay is different from the newdot.

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [ ] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.69.0
**Reproducible in staging?:** y
**Reproducible in production?:** n
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/93399543/dd3f317f-8857-4543-b5ff-48f9c8aae89e

https://github.com/Expensify/App/assets/93399543/6b439acc-db9a-4d4b-a068-85f1adf34131

**Expensify/Expensify Issue URL:**
**Issue reported by:** @Krishna2323 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1694612443986939

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by mountiny** [comment]
This is from @rushatgabhane @adamgrzybowski but i dont think this is a blocker

**Comment by s-alves10** [comment]
This is a regression of [this PR](https://github.com/Expensify/App/pull/26385), but it looks by intention

**Comment by dangrous** [comment]
Yeah it seems intentional, in order to fix https://github.com/Expensify/App/issues/26405

@shawnborton can you give this a quick look to confirm what color for the overlay is expected here? You had looked at the PR, too, but not sure if your approval included the color, or just the size issue that was being fixed. Thank you!

**Comment by dangrous** [comment]
removing the label, shouldn't block deploy and will sync with Shawn once he's in working hours

**Comment by shawnborton** [comment]
Chatting in Slack but yeah, I think we messed up and the overlay color is now incorrect. We should standardize on the correct color and update it everywhere. I'm thinking maybe using our colors.highLightBG color @ 72% opacity. Which would be `background: rgba(7, 39, 31, 0.72);`

**Comment by shawnborton** [comment]
cc @dannymcclain

**Comment by dannymcclain** [comment]
Totally agree @shawnborton. Let's make it happen. 🚀

**Comment by ishpaul777** [comment]
## Proposal
### Problem
Overlay background colour is different 

### Root Cause
No root cause, just a color change

### Changes
We need to change `backgroundColor` to #07271FB8 and overlayCapacity to `0.72` in [overlayStyles](https://github.com/ishpaul777/App/blob/8fcb69eed005554030734060e6e56dee218d043a/src/styles/styles.js#L1545), ofc we will use the variables for color and overlay capacity.

<img width="1822" alt="Screenshot 2023-09-14 at 9 15 57 PM" src="https://github.com/Expensify/App/assets/104348397/916c9a16-a3ed-49ec-9d6d-d1971d95df9f">

**Comment by dannymcclain** [comment]
@ishpaul777 forgive me if this is a dumb comment/if I'm misunderstanding something, but the hex code `#07271FB8` includes opacity, right? So if we set that as the background color AND set the opacity to 72%, won't that give us an actual opacity of 51.84%? (72% of 72% = 51.84%)

Our `Highlight BG` hex code is `#07271F`. We want _that_ value at 72%. Just to be clear!

Just want to make sure we don't inadvertently update this to be wrong. Again, please forgive me if I am just not understanding something. 🙈
cc @shawnborton

**Comment by ishpaul777** [comment]
Hii @dannymcclain, thanks for correcting it was dumb mistake I made. Updated the Proposal.

**Comment by dangrous** [comment]
I'm gonna add the Bug label to this so we can peruse external proposals!

Shawn and Danny - it looks like from the Slack conversation that we're going to likely want different opacities depending on the theme. Should we add that as a separate theme variable, or include it in a specific separate color variable for the overlay in the theme?

**Comment by shawnborton** [comment]
@grgia can you provide guidance on how we should handle adding a new color variable for the modal overlay BG to both dark and light themes? Or do we just get the new dark theme value merged here and we'll deal with light mode separately?

**Comment by grgia** [comment]
We could use a helper function to set the rgba for a hex in FE. I _think_ I've seen that pattern in app for some of these overlays. As part of this issue, we should make sure that all overlays in App reference the same variable/method

**Comment by s77rt** [comment]
@adamgrzybowski @rushatgabhane Would you like to handle this? As it's within the regression period of your PR. Otherwise we can assign @ishpaul777

**Comment by ishpaul777** [comment]
I can raise a PR within a hour. @s77rt

**Comment by dangrous** [comment]
> @adamgrzybowski @rushatgabhane Would you like to handle this? As it's within the regression period of your PR. Otherwise we can assign @ishpaul777

Bumping this, it's probably easiest if y'all can grab this?

**Comment by dangrous** [comment]
@s77rt I know @rushatgabhane is working on another project now so maybe we can assign @ishpaul777 for expediency's sake, as long as the proposal still sounds good to you?

@grgia I think you're referring to getBackgroundColorWithOpacityStyle, right? And so we could store both the theme color and the theme opacity in the theme somehow. Does that make sense? I'm not sure if there's anything tricky about adding to a theme or if we can just, er, add it.

**Comment by s77rt** [comment]
@dangrous That sounds good to me. 

Just I don't think we would need `getBackgroundColorWithOpacityStyle` we just have to change two lines:

https://github.com/Expensify/App/blob/a190e80086fcc964287cb1feb395f72eb29bb392/src/styles/styles.js#L1556


https://github.com/Expensify/App/blob/5f06b366dfa10976fe2d2aaf508099859e1da4a5/src/styles/variables.ts#L91

**Comment by shawnborton** [comment]
Just a heads up to whoever is implementing this - I think we're going to be changing the overlay color just slightly and this is a change that should happen globally, everywhere. Do we have an existing PR for this or are we still taking proposals?

**Comment by s77rt** [comment]
@shawnborton We will use `theme.overlay` so we should be using whatever value is defined for the overlay (currently it's `colors.darkHighlightBackground` for dark theme).

I think we are just waiting for @ishpaul777 to be assigned here. cc @dangrous

**Comment by shawnborton** [comment]
Okay, the updated values:
- Dark mode Overlay BG: `rgba(26, 61, 50, 0.72)`
- Light mode Overlay BG: `rgba(235, 230, 223, 0.72)`

**Comment by s77rt** [comment]
Are we supposed to change the `darkHighlightBackground` value in this case or create a new color?

**Comment by shawnborton** [comment]
This has nothing to do with darkHighlightBackground, so no need to touch that.

**Comment by dangrous** [comment]
Assigned! Let's make sure any values that we update live inside the themes!

**Comment by s77rt** [comment]
- The PR that introduced the bug has been identified: https://github.com/Expensify/App/pull/26385
- The offending PR has been commented on: https://github.com/Expensify/App/pull/26385#discussion_r1341733319
- A discussion in #expensify-bugs has been started: I don't think this is needed here
- Determine if we should create a regression test for this bug: No

**Comment by dangrous** [comment]
Yeah this was not a bug that would have been caught by any process I don't think - we should be good!

**Comment by laurenreidexpensify** [comment]
Payment Summary: 

-  External issue reporter @Krishna2323  - $50 - payment issued in Upwork
-  Contributor that fixed the issue @ishpaul777 - $500 + 50% urgency bonus = $750, payment issued in Upwork 
-  Contributor+ that helped on the issue and/or PR - @s77rt - $500 + 50% urgency bonus = $750, payment issued in Upwork

**Comment by s77rt** [comment]
@laurenreidexpensify The payment here should be $250 as the PR caused a regression no bonus is applicable and the total is subject to -50%.
